src/Pages/AllRecipes/index.tsx

```diff
@@ -1,9 +1,43 @@
+import { useContext, useEffect, useState } from "react";
 import Header from "../../Components/Header";
+import BASE_URL from "../../settings";
+import UserContext from "../../Contexts/UserContext";
+import RecipeCard from "../../Components/RecipeCard";
+import { Link } from "react-router";
+import Button from "../../Utilities/Button";
+
+
+interface Recipes {
```

Comment:
This should be called `RecipeSummary` or something because that's what it is. Recipes implies it is an array or some kind of collection.

Reply:
Working on this